test/e2e/e2eutil.go

```diff
@@ -181,6 +181,10 @@ func NewTestMongoDB(name string, namespace string) (mdbv1.MongoDB, mdbv1.MongoDB
 							DB:   "admin",
 							Name: "clusterAdmin",
 						},
+						{
+							DB:   "admin",
+							Name: "userAdmin",
```

Comment:
This was necessary in order to allow the new e2e test to see roles (none of the other roles for this user have the `rolesInfo` privilege). It seemed easier than adding a flag/new parameter to this function. It might make more sense to try to use a custom role to do this. Thoughts?